addons/link_tracker/tests/test_merge_utm_campaign.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tools import mute_logger
+from odoo.tests.common import TransactionCase
+
+
+class TestMergeUtmCampaign(TransactionCase):
+    def setUp(self):
+        result = super(TestMergeUtmCampaign, self).setUp()
```

Comment:
No need to save the result and return it, `setUp` doesn't return anything originally.